src/components/CardGroup/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { ChakraProps, SimpleGrid } from '@chakra-ui/react'
+import { Card } from '@components'
+
+interface CardGroupProps extends ChakraProps {
+  items: SubpageType[]
+  isSimple: boolean
+}
+
+export const CardGroup = ({ items, isSimple }: CardGroupProps): JSX.Element => {
+  return (
+    <SimpleGrid columns={{ base: 1, md: 2, lg: 4 }} spacing={10}>
+      {items.map((item, i) => (
+        <Card key={i} item={item} isSimple={isSimple} />
```

Comment:
I've noticed that `page.subpages[<index>].page` comes as id number instead of populated page data. That's why card links don't work.

In Card component `item.page.slug` is `undefined` and our card links look like `/en/undefined/first-event`.

Let's ignore this issue and we can solve it later.